api/organisations/subscriptions/licensing/__init__.py

```diff
@@ -0,0 +1 @@
+# TODO: split this into a private package?
```

Comment:
Ah yes, good point, we probably should do this... 🤔

Reply:
That was actually one of your TODOs, not mine. I'm not convinced it is worth the trouble to splitting this into a private package.